patchwork-capabilities/src/main/java/net/minecraftforge/common/capabilities/CapabilityDispatcher.java

```diff
@@ -59,34 +60,34 @@ public CapabilityDispatcher(Map<Identifier, ICapabilityProvider> providers, List
 
 	@SuppressWarnings("unchecked")
 	public CapabilityDispatcher(Map<Identifier, ICapabilityProvider> providers, List<Runnable> listeners, @Nullable ICapabilityProvider parent) {
-		List<ICapabilityProvider> lstCaps = Lists.newArrayList();
-		List<INBTSerializable<Tag>> lstWriters = Lists.newArrayList();
-		List<String> lstNames = Lists.newArrayList();
-		this.listeners = listeners;
+		List<ICapabilityProvider> capabilities = Lists.newArrayList();
```

Comment:
Just to match the field name.
```suggestion
		List<ICapabilityProvider> providers = Lists.newArrayList();
```

Reply:
And what would the first parameter of this method be then?

Reply:
`providerMap`?